docs/hopr-documentation/docs/developers/starting-local-cluster.md

```diff
@@ -25,6 +25,15 @@ to quickly setup everything for you to get started.
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/hoprnet/hoprnet)
 
+After Gitpod creates a virtual machine with our codebase, it will immediately start running a local cluster as described by our
+[Gitpod configuration file](https://github.com/hoprnet/hoprnet/blob/master/.gitpod.yml). The entire setup will take roughly 5-10
+minutes, after which it will `export` a series of endpoint URLs which you can use later.
+
+```bash
+gitpod /workspace/hoprnet (master) $ echo $HOPR_NODE_1_HTTP_URL
```

Comment:
You can also mention `$HOPR_NODE_1_ADDR` and `$HOPR_NODE_1_WS_URL`

Reply:
Ah good point, let me add those.